client/src/components/OrderPropertyNames/OrderPropertyNames.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import styles from './OrderPropertyNames.module.scss';
+
+interface IProps {}
+
+export const OrderPropertyNames: React.FC<IProps> = ({}) => {
+    const renderColumnName = (text: string) => {
+        return (
+            <div>
+                <div className={styles.nameItem}>{text}</div>
+            </div>
+        );
+    };
+
+    const renderNamesBlock = () => {
+        return (
+            <div className={styles.namesBlock}>
+                {renderColumnName('id')}
```

Comment:
Btw can be written like that 

const columns = ['id', 'order items', 'address', 'phone', 'status', 'total price']
{columns.map(renderColumnName)}